pkgs/top-level/python-packages.nix

```diff
@@ -5184,6 +5184,24 @@ in {
     };
   };
 
+  pytest-datafiles = buildPythonPackage rec {
```

Comment:
as mentioned in the header of the file as well as in the docs, this should be in a separate file